app/src/i18n/locales/en/data.json

```diff
@@ -34,6 +34,7 @@
   "add-data-details": "Manually add your calculated values for required sub-sectors to the inventory.",
   "select-subsector": "Select sub-sector",
   "residential-buildings": "Residential buildings",
+  "residential-buildings-description": "(needs translation)",
```

Comment:
Let's get these values from either the data or product teams. I guess we'll need one for every sector, subsector and subcategory (there is a TODO text on those in the manual data entry form).